src/js/parsers/section.js

```diff
@@ -341,9 +351,11 @@ class SectionParser {
 
   _createMarker() {
     let { state } = this;
-    let text = transformHTMLText(state.text);
-    let marker = this.builder.createMarker(text, state.markups);
-    state.section.markers.append(marker);
+    if (state.section.isMarkerable) {
```

Comment:
Should this check be in `parseElementNode`? https://github.com/bustle/mobiledoc-kit/blob/6f57a62f5979339a9f5a6e273bb5d58d1cc4681d/src/js/parsers/section.js#L238 It seems less than ideal that `_createMarker` has an internal guard to sometimes *not* do what it is presumed to do. I think we should be catching this a bit higher up.

Reply:
👍 updated to move checks to where we're calling `_createMarker`